### ISSUE TITLE
Indicate if the angles of selected curve points are the same.

### ISSUE THREAD

**Issue by LettError** [typesupply/glyph-nanny#28]
For instance when comparing flex points at the center of an S. Select both, see if they're parallel.

**Comment by typesupply** [comment]
Good idea. It could note any angles that are close to, but not exactly, parallel.

**Comment by adrientetar** [comment]
It's what I do in a tool that derives from that code, checking exactly equal angles generally only works for h/v segments, afaict.
